cpp/QuadratureRule.h

```diff
@@ -59,8 +65,9 @@ class QuadratureRule
   /// Return the number of quadrature points per entity
   std::size_t num_points(int i) const;
 
-  /// Return the topological dimension of the quadrature rule
-  std::size_t tdim() const { return _tdim; };
+  /// Topological dimension of the quadrature rule
```

Comment:
I guess you can rephrase this to the dimension of the quadrature points i.e. x=x_i, i=0,…,tdim-1